src/components/ConfigureSupersetEmbeddedDashboardModal/UpdateSupersetEmbeddedDashboard.js

```diff
@@ -0,0 +1,219 @@
+                                <NoticeBox
+                                    error
+                                    title={i18n.t(
+                                        'Could not load dashboard details'
+                                    )}
+                                >
+                                    {parseErrorText(queryError)}
+                                </NoticeBox>
+                            )}
+                        </div>
+                    )}
+                    <SupersetEmbeddedDashboardFields
+                        isSupersetEmbedIdValid={isSupersetEmbedIdValid}
+                        isSupersetEmbedIdFieldTouched={
+                            isSupersetEmbedIdFieldTouched
+                        }
+                        values={values}
+                        onChange={onChange}
+                        onSupersetEmbedIdFieldBlur={onSupersetEmbedIdFieldBlur}
+                        submitting={mutationLoading || queryLoading}
+                    />
+                    {mutationError && (
+                        <NoticeBox error title={mutationErrorTitle}>
+                            {mutationErrorText}
+                        </NoticeBox>
+                    )}
+                </ModalContent>
+                <ModalActions>
+                    <div className={styles.buttonStrip}>
+                        {!queryError && (
+                            <Button
+                                loading={mutationLoading}
+                                type="submit"
+                                primary
+                                disabled={
+                                    !hasFieldChanges ||
+                                    !isSupersetEmbedIdValid ||
+                                    queryLoading
+                                }
+                            >
+                                {i18n.t('Update dashboard')}
+                            </Button>
+                        )}
+                        <Button
+                            disabled={mutationLoading}
+                            secondary={!queryError}
+                            primary={!!queryError}
+                            onClick={closeModal}
+                            type={queryError ? 'submit' : undefined}
+                        >
+                            {i18n.t('Cancel')}
+                        </Button>
+                        {!queryError && dashboard?.access?.delete && (
+                            <Button
+                                destructive
+                                disabled={mutationLoading || queryLoading}
+                                secondary
+                                onClick={handleDelete}
+                                className={styles.deleteButton}
+                            >
+                                {i18n.t('Delete')}
+                            </Button>
+                        )}
```

Comment:
This is a little bit of lazyness on my side.... I should probably always show the button, but disable it with a tooltip explain that the user does not have access rights. Since this is not in the specs and because this modal shows after hitting an EDIT button I thought I could possible get away with it.